lib/bigcommerce/connection.rb

```diff
@@ -125,6 +129,7 @@ def request_legacy(method, path, options, headers={})
       resource_options = {
           :user => @configuration[:username],
           :password => @configuration[:api_key],
+          :verify_ssl => @configuration[:verify_ssl] || OpenSSL::SSL::VERIFY_PEER,
```

Comment:
Yes, I tested the code with only `:verify_ssl => @configuration[:verify_ssl]`. In the case, where `:verify_peer` is not passed in API config, `@configuration[:verify_ssl]` would return `nil` and RestClient treats it as OpenSSL::SSL::VERIFY_NONE. That's the reason for adding  `|| OpenSSL::SSL::VERIFY_PEER` part.